hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CreateTopLevelDagVisitor.java

```diff
@@ -315,8 +317,9 @@ public Vertex onSort(SortPhysicalRel rel) {
         Edge edge = between(sortVertex, combineVertex)
                 .ordered(comparator)
                 .distributeTo(localMemberAddress)
-                .allToOne(coordinatorPartitioningKey);
+                .allToOne("");
         dag.edge(edge);
+        this.partitionPruningLevel.add(COORDINATOR_REQUIRED);
```

Comment:
do you repetition here and easy chance to forget about it? For example you forgot about it in `IMapSqlConnector`

Reply:
you also forgot about it in `Joiner`

Reply:
3676d99b60fcb19aa514c834bd248ad827b41676 introduced too many workarounds and technical debt. It became evident that this is a dead end. So I changed the code to do the analysis on Jet side which is much simpler and less error prone.